Sources/ChessKitEngineCore/Engines/Extensions/stockfish+engine.cpp

```diff
@@ -3,38 +3,22 @@
 //  ChessKitEngine
 //
 
-#import "stockfish+engine.h"
-
-#import "../Stockfish/src/bitboard.h"
-#import "../Stockfish/src/endgame.h"
-#import "../Stockfish/src/evaluate.h"
-#import "../Stockfish/src/position.h"
-#import "../Stockfish/src/psqt.h"
-#import "../Stockfish/src/search.h"
-#import "../Stockfish/src/thread.h"
-#import "../Stockfish/src/uci.h"
-#import "../Stockfish/src/types.h"
+#include "stockfish+engine.h"
+#include "../Stockfish/src/_main.h"
+#include "../Stockfish/src/thread.h"
 
 using namespace Stockfish;
 
 void StockfishEngine::initialize() {
-    UCI::init(Options);
-    Tune::init();
-    PSQT::init();
-    Bitboards::init();
-    Position::init();
-    Bitbases::init();
-    Endgames::init();
-    Threads.set(size_t(Stockfish::Options["Threads"]));
-    Search::clear(); // After threads are up
-    Eval::NNUE::init();
+    char empty[] = "";
+    char* argv[] = { empty };
+    _main(1, argv);
 }
 
 void StockfishEngine::deinitialize() {
-    Threads.clear();
-    Threads.end();
+    ThreadPool().end();
 }
 
 void StockfishEngine::send_command(const std::string &cmd) {
-    UCI::execute_command(cmd);
+//    UCI::execute_command(cmd);
```

Comment:
Again, once lc0 works the same way this method probably can be removed altogether.